src/views/Activity.vue

```diff
@@ -6,14 +6,12 @@ div
 
   p(style="color: red;") {{ errormsg }}
 
-  h4(style="color: red;")
-    | This is currently work in progress and is known to have issues (such as
-    a(href='https://github.com/ActivityWatch/aw-webui/issues/22')  needing a date selector
-    | ,
-    a(href='https://github.com/ActivityWatch/activitywatch/issues/72')  timezone issues
-    | , etc.) - see
-    a(href='https://github.com/ActivityWatch/aw-webui/issues')  all issues
-    | .
+  //div.alert.alert-info(role="alert")
```

Comment:
Why is this hidden?
I commented it out myself and the list seems to be badly positioned, but looks much better otherwise.
Also, I don't think you should remove the 'timezone issues' and 'all issues' links